docs/narr.rst

```diff
@@ -571,16 +571,84 @@ There is a ``condition`` attribute in the
 "http://namespaces.zope.org/zcml" namespace which is honored on all
 elements in ZCML.  The value of the attribute is an expression
 which is used to determine if that element and its descendents are
-used.  If the condition is true, processing continues normally,
+used.  If the condition evaluates to true, processing continues normally;
 otherwise that element and its descendents are ignored.
 
-Currently the expression is always of the form "have featurename", and it
-checks for the presence of a ``<meta:provides feature="featurename" />``.
+The expression is always of the form "*verb* *arguments*". There are
+four verbs that are supported:
+
+- have
+- not-have
+- installed
+- not-installed
+
+Note that logical "if/else" conditions can be achieved by using both the
+positive verb and its not-prefixed counterpart on sibling elements
+with the same arguments. Logical "and" conditions can be achieved by
+nesting elements. To group arbitrary directives under a condition,
+nest them under a new ``<configure>`` element.
+
+.. seealso:: :meth:`zope.configuration.xmlconfig.ConfigurationHandler.evaluateCondition`
+
+   For documentation and examples of the XML implementation of conditions.
+
+Features
+--------
+
+The verbs ``have`` and ``not-have`` test for the presence or absense
```

Comment:
Typo:  `absence`

Reply:
Thanks. My spell-checker doesn't flag that for some reason...